ptrail/regularization/goemetric_sinuosity.py

```diff
@@ -0,0 +1,188 @@
+"""
+    | Author: Abhishek Gujjar
+"""
+
+import pandas as pd
+from typing import Union
+
+
+def traj_sinuosity_wrapper(trj) -> Union[float, None]:
+    """
+    Wrapper function to calculate TrajSinuosity for a trajectory.
+
+    Args:
+        trj: Input trajectory
+
+    Returns:
+        TrajSinuosity value or None if an error occurs
+    """
+    try:
+        return TrajSinuosity(trj)
+    except:
+        return None
+
+
+# Calculate geometric_sinuosity for each trajectory
+geometric_sinuosity = [traj_sinuosity_wrapper(trj) for trj in trajectories]
```

Comment:
Do not do testing like this inline in the source code file, this is not what is done in the library. Please see how other modules are coded and tested. Make a new jupyter notebook show all your testing there.